torchao/quantization/quant_primitives.py

```diff
@@ -838,6 +840,122 @@ def _choose_qparams_affine(
     return scale.to(dtype=scale_dtype), zero_point
 
 
+def choose_qparams_and_quantize_affine_qqq(
+    w: torch.Tensor,
+    mapping_type: str,
+    num_bits: int,
+    group_size: int,
+) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+    assert (
+        mapping_type == MappingType.SYMMETRIC
```

Comment:
seems like a dupe of the implementation of the stuff in utils can we directly call that func in here/ is this needed?

Reply:
have removed the stuff in utils and call the func here.